### DISCUSSION TITLE
Running Magpie on a separate GPU?

### DISCUSSION THREAD

**Discussion by Naizuri77** [Blinue/Magpie#80] [General]
Has anyone with multiple GPUs (either a dedicated GPU and an iGPU or two dedicated GPUs) tried to run Magpie on a separate GPU from the one rendering the game in order to not lose performance when upscaling? My system doesn't has an iGPU, so I can't try it myself unless I install a second GPU.

From Windows you can select which GPU a program will use, so in theory I think it should be possible to do the upscaling on a different GPU from the one rendering the game.

**Comment by zaifan** [comment]
I did, i ran magpie with intel uhd 630, so magpie didn't eat my dedicated gpu power (gtx 1050 4 gb laptop). Magpie ran well with this config. Magpie took igpu resource (usage) about 50% for 60 fps gameplay and 25% igpu resource for 30 fps gameplay.

**Reply by Naizuri77** [reply]
Thanks, that's good to know. May not be worth the trouble of installing my old GPU, but that's very useful for anyone with an integrated GPU. This would also make all the upscalers usable for 3D games and not just FSR.
